homeassistant/components/emoncms/sensor.py

```diff
@@ -84,20 +86,19 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the Emoncms sensor."""
-    apikey = config.get(CONF_API_KEY)
-    url = config.get(CONF_URL)
+    apikey = str(config.get(CONF_API_KEY))
+    url = str(config.get(CONF_URL))
```

Comment:
This doesn't make sense:
- if the key is always there, use `config[CONF_API_KEY]`
- if the key is optional, use `apikey: str | None = config.get(CONF_API_KEY)`

Same for url

Reply:
done